website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,105 @@
+
+When choosing a project to contribute to, it's important to consider your interests and skill level. Look for projects that align with your strengths and that you're passionate about. CNCF projects have a large and supportive community, with many resources available for contributors. It is very important to start by reading the project documentation and exploring the issue tracker. If you're new to the project, look for issues labeled as "good first issue" or "help wanted" - these are often small, well-defined tasks that are a good place to start.
+
+### Understand the Project
+Using an open-source project before becoming a contributor is an essential step to understand the project's functionality and architecture. If available, follow the project’s tutorial or getting started documentation and then try it out yourself to become familiar with how the project works and its capabilities. It helps contributors identify issues and challenges that the project faces, allowing them to get familiar with the codebase and the project's design principles. By testing the project, contributors can provide valuable feedback to the development team, which can help improve the project's quality and usability. Additionally, testing the project can help contributors identify areas where they can contribute to the project, allowing them to make more significant contributions and become more effective contributors.
+
+### Read the Documentation
+Before you start contributing to a project, it's important to read the documentation. Most projects have a README file that provides an overview of the project, how to set it up, and how to contribute. Read the documentation carefully to make sure you understand the project's goals, coding standards, and contribution guidelines. 
+
+It's also important to invest time in reading the project's mailing list threads, community forum discussions, and meeting minutes, as well as reviewing open and closed issues and pull requests in the project repository. By doing so, you can gain a deeper understanding of the project's context, goals, and challenges, and develop a sense of the community's culture and values. This knowledge can help you ask more informed and relevant questions when reaching out to the community, and increase your chances of getting a helpful and constructive response.
```

Comment:
Hmm, maybe taking a look at the roadmap can be mentioned here instead of the section above.

Reply:
Same here, get back to this when we have a roadmap